matchmaker/views/external_api.py

```diff
@@ -198,9 +197,7 @@ def _generate_notification_for_incoming_match(results, incoming_query, incoming_
             )
             email_message.send()
         except Exception as e:
-            email_errors.append(str(e))
-    if email_errors:
-        raise ValueError('\n'.join(email_errors))
+            logger.error(f'Unable to send notification email for incoming MME match with {submission_id}: {e}')
```

Comment:
since we're no longer re-raising a `ValueError` and logging instead we may never reach the original unclear error [here](https://github.com/broadinstitute/seqr/pull/3961/files#diff-45104ab8be01f23b15bb10bfdd314898603426372f37d7c412a50c93f713017fR85), I'd get rid of the original error unless logging the patient ID in addition to submission ID is also helpful

Reply:
patient ID is actually derived form submission ID, so agreed that thats not necessary anymore

Reply:
however, we do want to keep they try/except because we want to be 100% sure that the end user gets a 200 response in this situation. But I can update that error to be clearer

Reply:
okay I could not figure out a way to get any of the rest of the code to raise an exception, so in the end its fine to remove it